algorithms/src/fft/evaluations.rs

```diff
@@ -38,7 +38,8 @@ pub struct Evaluations<F: PrimeField> {
 
 impl<F: PrimeField> Evaluations<F> {
     /// Construct `Self` from evaluations and a domain.
-    pub fn from_vec_and_domain(evaluations: Vec<F>, domain: EvaluationDomain<F>) -> Self {
+    pub fn from_vec_and_domain(mut evaluations: Vec<F>, domain: EvaluationDomain<F>) -> Self {
+        evaluations.resize(domain.size(), F::zero());
```

Comment:
perhaps a comment would be useful here? I assume these extra zeros are not meaningful and only serve as padding?